cql/DashboardLibrary.cql

```diff
@@ -1368,9 +1368,9 @@ define ErrorDiagnosticReportWithNoConclusonCodes:
       )
     ) or
     (
-      Exists(HistologyDiagnosticReports) and
+      Exists(AllCompletedDiagnosticReport) and
```

Comment:
Your PR comment mentions: "_This is a slight performance degradation - we should find a way to not have to loop over AllCompletedDiagnosticReports twice to accomplish this._"

Can you open Jira ticket if this requires further performance fixes.

Reply:
Assuming you mean [CCSMCDS-361](https://jira.mitre.org/browse/CCSMCDS-361)?